utility/getPipelineStages.py

```diff
@@ -0,0 +1,135 @@
+import json
+import logging
+import os
+import random
+import string
+import sys
+
+import yaml
+from docopt import docopt
+
+log = logging.getLogger(__name__)
+doc = """
+This script fetches all the tests to be run for a pipeline based on the RHCS version and overrides
+
+    Usage:
+        utility/getPipelineStages.py --rhcephVersion <VER> --tags <tags>
+                  [--overrides <str>]
+                  [--metadata <FILE>]
+
+        utility/getPipelineStages.py (-h | --help)
+
+    Options:
+        -h --help          Shows the command usage
+        -v --rhcephVersion VER     The rhcephVersion for which test stages need to be fetched
+        -t --tags <str>    tags to be used for filtering test scripts
+        -o --overrides <str>       Overrides to be considered for execution
+        -m --metadata FILE         Metadata file full path if it is not present in default pipeline/metadata folder
+"""
+
+
+def load_file(file_name):
```

Comment:
why separate function for loading file. This can be simply written in more pythonic way -
```
content = yaml.safe_load(open(file_path, "r"))
```

Reply:
Done.